src/bundle/Resources/public/scss/ui/modules/muti-file-upload/_progress.bar.scss

```
@@ -1,38 +1,35 @@
 .c-progress-bar {
-    background: #d8d8d8;
-    height: calculateRem(6px);
     width: 100%;
-    border-radius: calculateRem(4px);
-    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
 
     &__value {
-        height: 100%;
-        background: $ibexa-color-primary;
+        background: linear-gradient(90deg, $ibexa-color-warning 0%, $ibexa-color-accent 44.95%, $ibexa-color-success 100%);
         border-radius: calculateRem(4px);
         transition: width 0.2s linear;
-    }
+        height: calculateRem(8px);
+        width: 20vh;
```

Shouldn't it be? It's strange that you set width relative to page height
```suggestion
        width: 20vw;
```